src/System.Private.CoreLib/shared/System/Runtime/InteropServices/CriticalHandle.cs

```diff
@@ -99,7 +95,7 @@ private MyCriticalHandleSubclass() : base(IntPtr.Zero)
           get { return handle == IntPtr.Zero; }
       }
 
```

Comment:
Not for this PR, but I guess at some point we can remove all these Reliability Contract things.